lib/codecs/src/decoding/mod.rs

```diff
@@ -260,6 +261,9 @@ pub enum DeserializerConfig {
         /// Apache Avro-specific encoder options.
         avro: AvroDeserializerOptions,
     },
+
+    /// TODO
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏This is rendering in the cue